tasks/identity.yml

```diff
@@ -4,22 +4,24 @@
     name: conjur
     state: present
 
-- name: Place Conjur public SSL certificate
-  copy:
-    dest: /etc/conjur.pem
-    content: "{{conjur_ssl_certificate}}"
-    mode: 0644
+- block:
+  - name: Place Conjur public SSL certificate
+    copy:
+      dest: "{{conjur_ssl_certificate_path}}"
+      content: "{{conjur_ssl_certificate}}"
+      mode: 0644
```

Comment:
What happens when "conjur_ssl_certificate_path" is not supplied? Don't we need here a "when"?